src/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function basicExpectedWins(runsScored, runsAllowed) {
```

Comment:
This is a default export, which means that you are exporting the function as-is.
It's better to do something like

```
module.exports = {
  basicExpectedWins
}
```
Because this way, you won't be breaking your interface when you expand the library in the future. What I mean is this situation in the future that could happen;

```
// let's say you wrote a function advancedExpectedWins some time in the future.
// now you can add:
module.exports  = {
  basicExpectedWins,
  advancedExpectedWins
}
// what would have happened if you kept the default export as a function? 
```

basically we're circumventing breaking changes, and keeping backwards compatibility, by using a slightly different pattern here.